node/src/proxy_server/mod.rs

```diff
@@ -4055,7 +4113,12 @@ mod tests {
         system.run();
 
         TestLogHandler::new().exists_log_containing(
-            format!("Discarding DnsResolveFailure message for <unspecified server> from an unrecognized stream key {:?}", stream_key).as_str());
+            format!(
+                "Discarding DnsResolveFailure message for <unspecified_server> from an unrecognized stream key {:?}",
+                stream_key
+            )
+            .as_str(),
```

Comment:
We usually use the ampersand because it's small and decent... but if you insist on this...